java-checks/src/main/java/org/sonar/java/checks/OperatorPrecedenceCheck.java

```diff
@@ -136,7 +149,7 @@ public void visitBinaryExpression(BinaryExpressionTree tree) {
     Tree.Kind peek = stack.peek();
     Tree.Kind kind = getKind(tree);
     if (requiresParenthesis(peek, kind)) {
-      raiseIssue(tree);
+      raiseIssue();
```

Comment:
Raise the issue on the line of the operator, which can be accessed directly using `tree.operatorToken().line()`